tests/test_models.py

```diff
@@ -31,7 +31,14 @@ def test_simulation_runs(self):
         # With a recombination_map of None, we simulate a coalescent without
         # recombination in msprime, with no mutation.
         contig = stdpopsim.Contig()
-        samples = self.model.get_samples(*([2] * self.model.num_populations))
+        # Generate vector with 2 samples for each pop with sampling enabled
+        sample_count = []
+        for p in self.model.populations:
+            if(p.allow_samples):
```

Comment:
Parentheses are unnecessary (also elsewhere)

Reply:
is this going to conflict with https://github.com/popgensims/stdpopsim/pull/187?